web/server/README.md

```diff
@@ -57,6 +57,35 @@ The API requests are serviced as follows:
 - `badges` gives access only to the badges API calls.
 - `management` gives access only to the management API calls.
 
+### Enabling TLS support
+
+## Certificate and key
+Netdata since version 1.16 has the feature to encrypt connections. To enable this option it is necessary to have a certificate and a key, case you do not have one yet, it is possible to generate them  with the following command
+
+$ openssl req -newkey rsa:2048 -nodes -sha512 -x509 -days 365 -keyout key.pem -out cert.pem
```

Comment:
As the `netdata` user i guess?
Does `netdata` user have enough permissions (write) to create certs in `ssl/` ?

Reply:
The ideal case it is the netdata user, but case the user installed the software with other user either it is necessary to use it or to give read permission for all users.

Reply:
I will state that the files need to be readable by netdata in the update I'll make tomorrow.